cms/djangoapps/contentstore/views/course.py

```diff
@@ -1338,12 +1372,18 @@ def group_configurations_list_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             group_configuration_url = reverse_course_url('group_configurations_list_handler', course_key)
             course_outline_url = reverse_course_url('course_handler', course_key)
-            configurations = GroupConfiguration.add_usage_info(course, store)
+            should_show_experiment_groups = should_show_experiment_group_configurations(course)
+            experiment_group_configurations = GroupConfiguration.get_split_test_partitions_with_usage(course, store)
```

Comment:
I'd change one or other name to match. I think I like split_test_partitions so that it matches the code's terminology rather than the UX. However, this entire class is called GroupConfiguration, so maybe it should all go with the UX terminology.